README.md

```diff
@@ -29,4 +29,10 @@ The more code you will read the more ideas can be received and the more potentia
 
 - [TannerLinsley.com](https://tannerlinsley.com) ([source](https://github.com/tannerlinsley/tannerlinsley.com))
 - [Dan Webb - Personal Website](https://danwebb.co) ([source](https://github.com/DanWebb/danwebb.co))
-- [Fourth Drive - Music Artist](https://fourthdrive.com)  ([source](https://gitlab.com/galmargalit1/fourth-drive))
\ No newline at end of file
+- [Fourth Drive - Music Artist](https://fourthdrive.com)  ([source](https://gitlab.com/galmargalit1/fourth-drive))
+
+### Not a React
```

Comment:
I think you need to change the name of this section to something like "Pure JS" and make it a second-level heading

Reply:
Ok, renamed to "Pure web", because this site doesn't use only JS.

Reply:
Thanks, looks good. Merged this PR